src/libraries/System.ComponentModel.Annotations/tests/System/ComponentModel/DataAnnotations/CompareAttributeTests.cs

```diff
@@ -74,6 +74,34 @@ public static void Validate_LowerAndUpperPropertyName_Success()
             Assert.Equal(nameof(CompareObject.comparepropertycased), attribute.OtherPropertyDisplayName);
         }
 
+        [Fact]
+        [SkipOnTargetFramework(~TargetFrameworkMonikers.NetFramework)]
+        public static void Validate_IncludesMemberName_NetFx()
```

Comment:
The change to the implementation does not appear to affect netfx. I tailored these tests based on this change: https://github.com/dotnet/runtime/commit/bf6b3a3db8b13110fbb4c34ecd58cfe506d5de60

Reply:
This looks weird to me. I thought we were trying to get away from functionality that was different between .NetFX and others? Also the commit https://github.com/dotnet/runtime/commit/bf6b3a3db8b13110fbb4c34ecd58cfe506d5de60 replaced a `TargetParameterCountException` with an `ArgumentException` which looks like a breaking change to me?

@ajcvickers what do you think?

Reply:
@pranavkm @lajones The behavior in .NET Framework will not change, since this is the core Code and we don't intend to port this change back.

@lajones That commit was merged some time ago; I don't think it is worth revisiting now.